content/docs/use-cases/sharing-data-and-model-files.md

```diff
@@ -9,7 +9,7 @@ supports Amazon S3, Microsoft Azure Blob Storage, Google Drive, Google Cloud
 Storage, SSH, HDFS, and other remote locations. The list is constantly growing.
 (For a complete list and configuration instructions, refer to `dvc remote add`.)
 
-![](/img/model-sharing-digram.png)
+![](/img/model-sharing-digram.png) _DVC enables independent data storage_
```

Comment:
why is it independent? no sure I understand where does it come from to be honest

Reply:
Just trying to describe the figure. The datas storage being independent (from code storage) is what enables sharing it easily, no? We can do any other caption, or remove it if you prefer. Doing this mainly for consistency so no strong op.

Reply:
yeah, I understand the idea ... it just sounds strange ...  it should be more like `DVC stores data in your storage`, or `Data is stored in your storage, separately from code`

Reply:
Done!